app/src/main/java/com/cornellappdev/android/eatery/CtownMenuActivity.java

```diff
@@ -150,16 +147,13 @@ public void onOffsetChanged(AppBarLayout appBarLayout, int verticalOffset) {
         mCafeRating.setRating(Float.parseFloat(((CollegeTownModel) mCafeData).getRating()));
 
         mCafeDirections = findViewById(R.id.cafe_directions);
-        mCafeDirections.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                // Opening up google maps with the latitude and longitude on click
-                String uri = String.format(Locale.ENGLISH,
-                        "http://maps.google.com/maps?daddr=%f,%f",
-                        mCafeData.getLatitude(), mCafeData.getLongitude());
-                Intent intent = new Intent(Intent.ACTION_VIEW, Uri.parse(uri));
-                startActivity(intent);
-            }
+        mCafeDirections.setOnClickListener((View v) -> {
+            // Opening up google maps with the latitude and longitude on click
+            String uri = String.format(Locale.ENGLISH,
+                    "http://maps.google.com/maps?daddr=%f,%f",
```

Comment:
Should this string be stored as a variable?

Reply:
If you mean in res/values/strings I think the reason why strings should be stored there is so they can easily be translated so we can support multiple languages - urls dont actually need to be there